doc/api/assert.md

```diff
@@ -354,8 +354,9 @@ function func() {}
 const callsfunc = tracker.calls(func);
 callsfunc(1, 2, 3);
 
-assert.deepStrictEqual(tracker.getCalls(callsfunc),
-                       [{ thisArg: undefined, arguments: [1, 2, 3] }]);
+assert.deepStrictEqual(tracker.getCalls(callsfunc), [
+  { thisArg: undefined, arguments: [1, 2, 3] },
+]);
```

Comment:
Still a lot of unrelated changes, let me know if you need help to get rid of those